backend/satellite_tools/download.py

```diff
@@ -278,16 +282,31 @@ class ThreadedDownloader:
     def __init__(self, retries=3, log_obj=None, force=False):
         self.queues = {}
         comp = CFG.getComponent()
-        initCFG('server.satellite')
+        initCFG("server.satellite")
         try:
             self.threads = int(CFG.REPOSYNC_DOWNLOAD_THREADS)
         except ValueError:
-            initCFG(comp)
-            raise ValueError("Number of threads expected, found: '%s'" % CFG.REPOSYNC_DOWNLOAD_THREADS)
-        else:
-            initCFG(comp)
+            raise ValueError(
+                "Number of threads expected, found: '%s'" % CFG.REPOSYNC_DOWNLOAD_THREADS
+            )
+        try:
+            self.timeout = int(CFG.REPOSYNC_TIMEOUT)  # we could also use CFG.TIMEOUT
```

Comment:
Here I was not sure if this should be its own setting or if we want to re-use [timeout in rhn_server.conf](https://github.com/agraul/uyuni/blob/reposync-add-timeout-config/backend/rhn-conf/rhn_server.conf#L29).

I think the former, but I'm interested in what you are thinking.

Reply:
I'd also say we should keep this as a separate setting. Using the rhn config would be kind of surprising, especially since its use-case is the rpclib right now.